modules/unsupported/ysld/src/main/java/org/geotools/ysld/Band.java

```diff
@@ -0,0 +1,92 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2014-2016, Open Source Geospatial Foundation (OSGeo)
+ *    
```

Comment:
@jodygarnett I guess that comes down to that argument you had with Arne Kepp about the GWC headers.  I have no idea which is the correct way to do this to make the lawyers happy.